client/src/components/Lists/ActionList.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from 'react';
+import axios from '../../api/Axios';
+import wants from '../../api/Wanted';
+import Cards from '../Slide/Card';
+
+const ActionList = () => {
+
+    const [actionmovie, setActionMovie] = useState([]);
+
+    useEffect(() => {
+        axios.get(wants.getaction)
+        .then(response => {
+            setActionMovie(response.data.results)
+        })
+    }, []);
+
+  return (
+    <div>
+      <br></br>
```

Comment:
Why would you add 6 <br> tahgs, and read more from w3schools about <br> tags to use correctly, 
https://www.w3schools.com/tags/tag_br.asp#:~:text=The%20tag%20inserts%20a,it%20has%20no%20end%20tag.